docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-elasticsearch.md

```diff
@@ -0,0 +1,109 @@
+---
+id: using-existing-elasticsearch
+title: "Using Existing Elasticsearch"
+description: "Learn how to use an existing Elasticsearch instance in Camunda 8 Self-Managed deployment."
+---
+
+By default, the [Helm chart deployment](../deploy.md) creates a new elassticsearch instance, but it's possible to use an existing Elasticsearch instance either inside the same Kubernetes cluster or outside of it. This guide steps through using an existing Elasticsearch instance.
+
+## Connecting to existing Elasticsearch without a certificate
+
+Connecting to Elasticsearch through `http` is possible by modifying `global.elasticsearch.protocol`
```

Comment:
@hamza-m-masood This needs a rewording since it's the default behavior, the user doesn't need to set or modify anything.
Also, the key now is `global.elasticsearch.url.protocol`.

Reply:
reworded